core/arch/arch.c

```diff
@@ -914,7 +914,8 @@ arch_profile_exit()
 #endif /* WINDOWS_PC_SAMPLE */
 
 /* arch-specific atexit cleanup */
-void d_r_arch_exit(IF_WINDOWS_ELSE_NP(bool detach_stacked_callbacks, void))
+void
+d_r_arch_exit(IF_WINDOWS_ELSE_NP(bool detach_stacked_callbacks, void))
```

Comment:
(I think a clang-format bug in whatever version was used in the initial formatting resulting in the void on the same line.)

Reply:
Corrected.